sqlitecluster/SQLiteNode.cpp

```diff
@@ -1079,9 +1085,7 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
         if (peer->params["Permafollower"] == "true" && message.calc("Priority")) {
             STHROW("you're supposed to be a 0-priority permafollower");
         }
-        if (peer->params["Permafollower"] != "true" && !message.calc("Priority")) {
```

Comment:
why are we deleting this protection?

Reply:
Leaving it would require that we choose a value for "not yet synced hosts", which could be 1, but would then require additional changes to synching code, because two nodes can't be the same priority. IMO, this is the better/easier change. I don't think this protection is really something we need to have. For that matter, I don't think we need to enforce the previous test for "you're supposed to be a permafollower".

Reply:
Based on what do you think we don't need them? They are meant to protect against misconfigurations, we should keep those protections unless we have an explicit reason not to. I don't think we do. I think we should choose a different value for `not yet synced hosts` or we should not use priority to signify `not yet synced hosts` and instead do something else. There's no reason we have to use priority to signify this.

Reply:
K, I can do that, one last thing though. It's somewhat common that we have to reconfigure the cluster to allow the permafollower to join the cluster as a full peer. If we just did away with those protections, we wouldn't have to bounce the cluster, which would be a win, IMO, for a low reduction in safety.

Reply:
not bouncing the cluster for config changes is a whole separate conversation. there's many different ways to solve that problem